pkg/testsuite/mock/wallet.go

```diff
@@ -176,6 +237,38 @@ func (w *Wallet) TransactionID(alias string) iotago.TransactionID {
 	return w.Transaction(alias).MustID()
 }
 
+func (w *Wallet) Account(accountID iotago.AccountID) *AccountData {
+	w.mutex.RLock()
+	defer w.mutex.RUnlock()
+
+	acc, exists := w.accounts[accountID]
+	if !exists {
+		panic(ierrors.Errorf("account %s not registered in wallet", accountID.ToHex()))
```

Comment:
```suggestion
		panic(ierrors.Errorf("account %s not registered in wallet", accountID))
```

Nit: It's better to use `%s` with IDs that support aliasing (like block IDs, tx IDs, account IDs, ...) so the alias is printed rather than the hex string. Especially helpful in a testing context.